plugin/src/main/java/org/opensearch/ml/model/MLTaskState.java

```diff
@@ -33,5 +33,6 @@
     CREATED,
     RUNNING,
     COMPLETED,
-    FAILED
+    FAILED,
+    CANCELLED
```

Comment:
Have we used this CANCELLED status anywhere, or this is reserved for future API usage?

Reply:
This is a reserved state